saleor/graphql/schema.graphql

```diff
@@ -4018,6 +4022,39 @@ enum StaffMemberStatus {
   DEACTIVATED
 }
 
+type StaffNotificationRecipient implements Node {
+  user: User
+  active: Boolean
+  id: ID!
+  privateMeta: [MetaStore]!
+  meta: [MetaStore]!
+  email: String
+}
+
+type StaffNotificationRecipientCreate {
+  errors: [Error!]
```

Comment:
should I manually remove errors in schema, because I already include `ShopError`
and
`python manage.py get_graphql_schema > saleor/graphql/schema.graphql` automatically  adds `errors`

Reply:
We can't do that right now so ignore this my comment.